CHANGELOG.rst

```diff
@@ -1,6 +1,33 @@
 Changelog
 =========
 
+1.0.0 (2021-02-xx)
+------------------
+
+- [CHANGE] Arrow has **dropped support** for Python 2.7 and 3.5.
+- [CHANGE] There are multiple **breaking changes** with this release, please see the `migration guide <https://github.com/arrow-py/arrow/issues/832>`_ for a complete overview.
+- [CHANGE] Made humanize() granularity="auto" limits more accurate to reduce strange results.
+- [NEW] Added support for Python 3.9.
+- [NEW] Added a new keyword argument "exact" to ``span``, ``span_range`` and ``interval`` methods. This makes timespans begin at the start time given and not extend beyond the end time given, for example:
+
+..code-block:: python
+    >>> start = Arrow(2013, 5, 5, 12, 30)
+    >>> end = Arrow(2013, 5, 5, 17, 15)
+    >>> for r in arrow.Arrow.span_range('hour', start, end, exact=True):
+    ...     print(r)
+    ...
+    (<Arrow [2013-05-05T12:30:00+00:00]>, <Arrow [2013-05-05T13:29:59.999999+00:00]>)
+    (<Arrow [2013-05-05T13:30:00+00:00]>, <Arrow [2013-05-05T14:29:59.999999+00:00]>)
+    (<Arrow [2013-05-05T14:30:00+00:00]>, <Arrow [2013-05-05T15:29:59.999999+00:00]>)
+    (<Arrow [2013-05-05T15:30:00+00:00]>, <Arrow [2013-05-05T16:29:59.999999+00:00]>)
+    (<Arrow [2013-05-05T16:30:00+00:00]>, <Arrow [2013-05-05T17:14:59.999999+00:00]>)
+
+
+- [FIX] Fixed handling of maximum permitted timestamp on Windows systems.
+- [FIX] Corrections to French, German, Japanese and Norwegian locales.
+- [INTERNAL] Added type annotations throughout the codebase.
```

Comment:
I think this should be under [NEW] since it is a huge change. Like `[NEW] Arrow now natively supports PEP 484-style type annotations.`